great_expectations/datasource/data_connector/configured_asset_glue_catalog_data_connector.py

```diff
@@ -0,0 +1,140 @@
+
+    A ConfiguredAssetGlueCatalogDataConnector requires an explicit listing of each DataAsset you want to
+    connect to. This allows more fine-tuning, but also requires more setup. You will need define to the
+    database and table names.
+    """
+
+    def __init__(
+        self,
+        name: str,
+        datasource_name: str,
+        execution_engine: Optional[ExecutionEngine] = None,
+        assets: Optional[Dict[str, dict]] = None,
+        boto3_options: Optional[dict] = None,
+        batch_spec_passthrough: Optional[dict] = None,
+    ):
+        """
+        A DataConnector that requires explicit listing of AWS Glue Catalog tables you want to connect to.
+
+        Args:
+            name (str): The name of this DataConnector
+            datasource_name (str): Name of datasource that this DataConnector is connected to
+            execution_engine (ExecutionEngine): Execution Engine object to actually read the data
+            assets (dict): dict of asset configuration
+            boto3_options (dict): optional boto3 options
+            batch_spec_passthrough (dict): dictionary with keys that will be added directly to batch_spec
+        """
+        logger.warning(
+            "Warning: great_expectations.datasource.data_connector.ConfiguredAssetGlueCatalogDataConnector is "
+            "experimental. Methods, APIs, and core behavior may change in the future."
+        )
+        super().__init__(
+            name=name,
+            datasource_name=datasource_name,
+            execution_engine=execution_engine,
+            assets=assets,
+            batch_spec_passthrough=batch_spec_passthrough,
+        )
+
+        if boto3_options is None:
+            boto3_options = {}
+
+        try:
+            self._glue = boto3.client("glue", **boto3_options)
+        except (TypeError, AttributeError):
+            raise ImportError(
+                "Unable to load boto3 (it is required for ConfiguredAssetGlueCatalogDataConnector)."
+            )
+
+    @property
+    def execution_engine(self) -> SparkDFExecutionEngine:
+        return cast(SparkDFExecutionEngine, self._execution_engine)
+
+    def _update_data_asset_name_from_config(
+        self, data_asset_name: str, data_asset_config: Optional[dict]
+    ) -> str:
+        if data_asset_config is None:
+            data_asset_config = {}
+
+        data_asset_name_prefix: str = data_asset_config.get(
+            "data_asset_name_prefix", ""
+        )
+        data_asset_name_suffix: str = data_asset_config.get(
+            "data_asset_name_suffix", ""
+        )
+
+        data_asset_name = (
+            f"{data_asset_name_prefix}{data_asset_name}{data_asset_name_suffix}"
+        )
+        return data_asset_name
+
+    def _get_batch_identifiers_list_from_data_asset_config(
+        self,
+        data_asset_name: str,
+        data_asset_config: Optional[dict],
+    ) -> List[dict]:
+        return [{}]
```

Comment:
@lccasagrande How come we are not simply reusing this method from the parent class?  Thanks.

Reply:
I did a code refactoring and I am not using the **ConfiguredAssetSqlDataConnector** as parent class anymore.